core/clingutils/CMakeLists.txt

```
@@ -85,7 +85,13 @@ if (builtin_clang)
   set(CLANG_RESOURCE_DIR_STEM ${CMAKE_BINARY_DIR}/interpreter/llvm-project/llvm/${CMAKE_CFG_INTDIR}/lib/clang)
   set(CLANG_RESOURCE_DIR_VERSION ${LLVM_VERSION_MAJOR})
 else ()
-  set(CLANG_RESOURCE_DIR_STEM ${LLVM_LIBRARY_DIR}/clang)
+  # If Clang was installed standalone (in particular, not in the LLVM tree),
+  # use the CLANG_INSTALL_PREFIX variable to indicate where it is.
+  if(DEFINED CLANG_INSTALL_PREFIX)
+    set(CLANG_RESOURCE_DIR_STEM ${CLANG_INSTALL_PREFIX}/lib/clang)
+  else()
+    set(CLANG_RESOURCE_DIR_STEM ${LLVM_LIBRARY_DIR}/clang)
+  endif()
```

I think we can always use `CLANG_INSTALL_PREFIX`, it's defined by `ClangConfig.cmake`. We should also take into account `LLVM_LIBDIR_SUFFIX` (untested):
```suggestion
  set(CLANG_RESOURCE_DIR_STEM ${CLANG_INSTALL_PREFIX}/lib${LLVM_LIBDIR_SUFFIX}/clang)
```

---

Thanks for the suggestion! I'll test it

---

For `LLVM_LIBDIR_SUFFIX`, we'll also need https://github.com/root-project/root/pull/18567